sparse/core.py

```diff
@@ -362,6 +530,69 @@ def _reduce(method, *args, **kwargs):
         return self.reduce(method, **kwargs)
 
     def reduce(self, method, axis=None, keepdims=False, **kwargs):
+        """
+        Performs a reduction operation on this array.
+
+        Parameters
+        ----------
+        method : numpy.ufunc
+            The method to use for performing the reduction.
+        axis : Union[int, Iterable[int]], optional
+            The axes along which to perform the reduction. Uses all axes by default.
+        keepdims : bool, optional
+            Whether or not to keep the dimensions of the original array.
+        kwargs : dict
+            Any extra arguments to pass to the reduction operation.
+
+        Returns
+        -------
+        COO
+            The result of the reduction operation.
+
+        Raises
+        ------
+        ValueError
+            If reducing an all-zero axis would produce a nonzero result.
+
+        Notes
+        -----
+        This function internally calls :obj:`COO.sum_duplicates` to bring the array into
+        canonical form.
+
+        See Also
+        --------
+        numpy.ufunc.reduce : A similar Numpy method.
```

Comment:
If you're feeling energetic you might also link to these with intersphinx.  Here is a recent PR to dask that does this: https://github.com/dask/dask/pull/3050

Reply:
They already are linked. Well, most of them, anyway.

Reply:
It seems it auto-detects most of them. The ones it doesn't, we need to just add `:obj:` to.

Reply:
Ah, alright.  Glad to hear it